src/platform/user/authorization/components/RequiredLoginView.jsx

```diff
@@ -145,7 +149,7 @@ class RequiredLoginView extends React.Component {
   };
 
   render() {
-    return <div>{this.renderWrappedContent()}</div>;
```

Comment:
Since I'm working in this file, there is no reason to wrap all child content in a `div`. I'm sure this `div` was used because Fragments weren't a thing when it was written.